tools/delete_candidate.php

```diff
@@ -65,10 +83,53 @@
 $query = "SELECT ID, Test_name, CommentID FROM flag WHERE SessionID in (" . 
          implode(" , ", $sessions) . ")";
 $instruments = $DB->pselect($query, array()); 
-
+//delete from genomic_candidate_files_rel
+$DB->delete("genomic_candidate_files_rel", array("CandID" => $DCCID));
+echo "----------------------delete genomic_candidate_files_rel-------\n";
+//delete from genomic_sample_candidate_rel
+$DB->delete("genomic_sample_candidate_rel", array("CandID" => $DCCID));
```

Comment:
There might be records in the genomic_cpg table and there is a foreign key using sample_label